conference.md

```diff
@@ -6,16 +6,66 @@ title: "Conference"
+  <div class="event-meta">
+    {% include icons/ticket.svg %}
+    <p class="event-cost event-meta-item">$20-$100 sliding scale, <a href="#registration">registration details below</a>
+    </p>
+  </div>
+  <div class="event-meta">
+    {% include icons/calendar-o.svg %}
+    <p class="event-time event-meta-item"><strong>{{page.startTime | date:"%-I:%M %P "}}</strong> – <strong>{{page.endTime | date:"%-I:%M %P"}}</strong></p>
+  </div>
+  <div class="event-meta">
+    {% include icons/map-marker.svg %}
+    <p class="event-location event-meta-item">
+    {% if page.locationName %}
+      {{ page.locationName }},
+    {% endif %}    
+    {% if page.locationLink %}
+      <a href="{{page.locationLink}}" target="_blank">{{ page.location }}</a> <!--_-->
+    {% else %}
+      {{ page.location }}
+    {% endif %}
+    </p>
+  </div>
+</div>
+
+***
 
 What kinds of creative and critical engagement with technology practices can enable meaningful change when we do it with others?
 
-Our theme this year focuses on how we **Do It With Others (DIWO)** as opposed to just ourselves, building on the “distributed campaign for emancipatory, networked art practices,” instigated by UK-based Furtherfield (2006) as a response to the Do It Yourself (DIY) movement. We have seen a growing number of peer-to-peer, inclusive, and privacy-respecting projects mobilizing in favour of resilient, accessible, equitable communications over the internet in 2017. And yet many open questions remain. In the face of threats to the open internet, which tools and tactics will help us recognize the opportunities and challenges of this moment?
+Our theme focuses on how we **Do It With Others (DIWO)** as opposed to just ourselves as we build new network infrastructures and draws on the distributed campaign for emancipatory, networked art practices, instigated by UK-based Furtherfield (2006) as a response to the Do It Yourself (DIY) movement. We have seen a growing number of peer-to-peer, inclusive, and privacy-respecting projects mobilizing in favour of resilient, accessible, equitable communications over the internet in 2017. And yet many open questions remain. In the face of threats to the open internet, which tools and tactics will help us recognize the opportunities and challenges of this moment?
+
+## Program Highlights
+
+
+
+
+
+Check back here late-May for our full conference program!
+
+***
+
+## Registration
+
+We have three admission types: Organization Supporter, Supporter, and Reduced Cost. The true cost for us to host this event is $57 per person. Your support at or above that price allows us to put sponsorship money elsewhere and offer reduced cost admissions. If it is difficult to attend even with a reduced cost admission, please contact us at {{ site.email }}. We are able to sport a limited number of scholarships and also are looking for volunteers interested in helping at the event in return for complimentary admission.
+
+| Admission  | Price  | Details |
```

Comment:
`Admission`->`Admission Level`?
Guessing we'd want to reflect Brown Paper's labelling here.

Reply:
👍 done